examples/react/css/src/common.css

```
@@ -0,0 +1,61 @@
+/* Motion reduction */
+@media (prefers-reduced-motion: reduce) {
+  [class^='ais-'] {
+    transition: none !important;
+  }
+}
+
+/* Focus outline */
+a[class^='ais-']:focus-visible,
+button[class^='ais-']:focus-visible,
+input[class^='ais-']:focus-visible {
+  outline: 2px solid
+    rgba(var(--ais-primary-color-rgb), var(--ais-primary-color-alpha));
+  outline-offset: 2px;
+}
+
+/* Widgets */
+[class^='ais-'][class*='-showMore'] {
+  border-radius: calc(var(--ais-spacing) / 4);
+  font-size: 0.9em;
+  font-weight: var(--ais-font-weight-medium);
+  color: #fff;
+  padding: calc(var(--ais-spacing-half) * 0.75)
+    calc(var(--ais-spacing-half) * 1.25);
+  background-color: rgba(
+    var(--ais-primary-color-rgb),
+    var(--ais-primary-color-alpha)
+  );
+  margin-top: calc(var(--ais-spacing-half) * 1.5);
+  transition: background-color var(--ais-transition-duration)
+    var(--ais-transition-timing-function);
+  box-shadow: var(--ais-shadow-xs);
+}
+
+@media (hover: hover) {
+  [class^='ais-'][class*='-showMore']:not(
+      [class^='ais-'][class*='-showMore--disabled']
```

```suggestion
      [class^='ais-'][disabled]
```
Do we prefer that syntax?

---

Could there be a case where you would use the class without the actual attribute?

---

I'm not sure about that, do you know if it's consistent @sarahdayan?

---

@FabienMotte I'd imagine it's quite unlikely but thought just throw it out there.

Could we have multiple variants of the selector to cover _potential_ edge cases?

---

We _should_ have the `disabled` attribute present on all show more buttons. If we realize it isn't the case we can always fix it in InstantSearch, it's not breaking and is actually a good fix.

---

Thanks for the feedback, I'll go with `[disabled]` then